modules/gapi/src/backends/ov/govbackend.cpp

```diff
@@ -1054,15 +1163,19 @@ struct InferROI: public cv::detail::KernelTag {
 
         const auto &input_name = uu.params.input_names.at(0);
         const auto &mm = in_metas.at(1u);
-        GAPI_Assert(cv::util::holds_alternative<cv::GMatDesc>(mm));
-        const auto &matdesc = cv::util::get<cv::GMatDesc>(mm);
-
+        GAPI_Assert(cv::util::holds_alternative<cv::GMatDesc>(mm) ||
+                    cv::util::holds_alternative<cv::GFrameDesc>(mm));
         const bool is_model = cv::util::holds_alternative<ParamDesc::Model>(uu.params.kind);
         const auto &input_shape = is_model ? uu.model->input(input_name).get_shape()
                                            : uu.compiled_model.input(input_name).get_shape();
-        if (!isImage(matdesc, input_shape)) {
-            util::throw_error(std::runtime_error(
-                "OV Backend: InferROI supports only image as the 1th argument"));
+
+        if (cv::util::holds_alternative<cv::GMatDesc>(mm)) {
```

Comment:
What about `GFrameDesc`? This is the exact point of handling both `GMatDesc` / `GFrameDesc` (they're part of `GMetaArg`) in `isImage` function. E.g

```
static bool isImage(const cv::GMetaArg& meta, const ov::Shape& shape) {
    if (cv::util::holds_alternative<GFrameDesc>(meta)) {
        return true;
    }
    GAPI_Assert(cv::util::holds_alternative<GMatDesc>(meta));
    auto matdesc = cv::util::get<GMatDesc>(meta);
    return isImage(matdesc, shape);
}
```

Reply:
Fixed, thanks! https://github.com/opencv/opencv/pull/24938/commits/833bd890ef003a7c161f3a91c61b9c8d81df0e55